### ISSUE TITLE
Input Action asset window crashes when previously serialised control not found

### ISSUE THREAD

**Issue by m4ff** [Unity-Technologies/InputSystem#836]
```
LayoutNotFoundException: Cannot find control layout 'XRInputV1::WindowsMR0x045E::OpenVRControllerWindowsMR0x045E0x065B01Left'
UnityEngine.InputSystem.Layouts.InputControlLayout+Cache.FindOrLoadLayout (System.String name) (at Library/PackageCache/com.unity.inputsystem@0.9.5-preview/InputSystem/Controls/InputControlLayout.cs:1872)
UnityEngine.InputSystem.InputControlPath+ParsedPathComponent.ToHumanReadableString (System.String parentLayoutName, System.String& referencedLayoutName) (at Library/PackageCache/com.unity.inputsystem@0.9.5-preview/InputSystem/Controls/InputControlPath.cs:1047)
UnityEngine.InputSystem.InputControlPath.ToHumanReadableString (System.String path, UnityEngine.InputSystem.InputControlPath+HumanReadableStringOptions options) (at Library/PackageCache/com.unity.inputsystem@0.9.5-preview/InputSystem/Controls/InputControlPath.cs:187)
UnityEngine.InputSystem.Editor.BindingTreeItem.get_displayPath () (at Library/PackageCache/com.unity.inputsystem@0.9.5-preview/InputSystem/Editor/AssetEditor/InputActionTreeViewItems.cs:356)
UnityEngine.InputSystem.Editor.BindingTreeItem.AddTo (UnityEditor.IMGUI.Controls.TreeViewItem parent, UnityEditor.SerializedProperty bindingProperty) (at Library/PackageCache/com.unity.inputsystem@0.9.5-preview/InputSystem/Editor/AssetEditor/InputActionTreeViewItems.cs:401)
UnityEngine.InputSystem.Editor.ActionTreeItem.AddBindingsTo (UnityEditor.IMGUI.Controls.TreeViewItem parent) (at Library/PackageCache/com.unity.inputsystem@0.9.5-preview/InputSystem/Editor/AssetEditor/InputActionTreeViewItems.cs:327)
UnityEngine.InputSystem.Editor.ActionMapTreeItem.AddActionsTo (UnityEditor.IMGUI.Controls.TreeViewItem parent, System.Boolean addBindings) (at Library/PackageCache/com.unity.inputsystem@0.9.5-preview/InputSystem/Editor/AssetEditor/InputActionTreeViewItems.cs:187)
UnityEngine.InputSystem.Editor.ActionMapTreeItem.AddActionsAndBindingsTo (UnityEditor.IMGUI.Controls.TreeViewItem parent) (at Library/PackageCache/com.unity.inputsystem@0.9.5-preview/InputSystem/Editor/AssetEditor/InputActionTreeViewItems.cs:173)
UnityEngine.InputSystem.Editor.InputActionTreeView.BuildWithJustActionsAndBindingsFromMap (UnityEditor.SerializedProperty actionMapProperty) (at Library/PackageCache/com.unity.inputsystem@0.9.5-preview/InputSystem/Editor/AssetEditor/InputActionTreeView.cs:75)
UnityEngine.InputSystem.Editor.InputActionEditorWindow+<>c__DisplayClass24_0.<RebuildActionTree>b__1 () (at Library/PackageCache/com.unity.inputsystem@0.9.5-preview/InputSystem/Editor/AssetEditor/InputActionEditorWindow.cs:452)
UnityEngine.InputSystem.Editor.InputActionTreeView.BuildRoot () (at Library/PackageCache/com.unity.inputsystem@0.9.5-preview/InputSystem/Editor/AssetEditor/InputActionTreeView.cs:103)
UnityEditor.IMGUI.Controls.TreeView+TreeViewControlDataSource.FetchData () (at C:/buildslave/unity/build/Editor/Mono/GUI/TreeView/TreeViewControl/TreeViewControlDataSource.cs:53)
UnityEditor.IMGUI.Controls.TreeViewDataSource.ReloadData () (at C:/buildslave/unity/build/Editor/Mono/GUI/TreeView/TreeViewDataSource.cs:53)
UnityEditor.IMGUI.Controls.TreeView+TreeViewControlDataSource.ReloadData () (at C:/buildslave/unity/build/Editor/Mono/GUI/TreeView/TreeViewControl/TreeViewControlDataSource.cs:25)
UnityEditor.IMGUI.Controls.TreeViewController.ReloadData () (at C:/buildslave/unity/build/Editor/Mono/GUI/TreeView/TreeViewController.cs:288)
UnityEditor.IMGUI.Controls.TreeView.Reload () (at C:/buildslave/unity/build/Editor/Mono/GUI/TreeView/TreeViewControl/TreeViewControl.cs:105)
UnityEngine.InputSystem.Editor.InputActionEditorWindow.RebuildActionTree () (at Library/PackageCache/com.unity.inputsystem@0.9.5-preview/InputSystem/Editor/AssetEditor/InputActionEditorWindow.cs:456)
UnityEngine.InputSystem.Editor.InputActionEditorWindow.OnActionMapTreeSelectionChanged () (at Library/PackageCache/com.unity.inputsystem@0.9.5-preview/InputSystem/Editor/AssetEditor/InputActionEditorWindow.cs:425)
UnityEngine.InputSystem.Editor.InputActionTreeView.SelectionChanged (System.Collections.Generic.IList`1[T] selectedIds) (at Library/PackageCache/com.unity.inputsystem@0.9.5-preview/InputSystem/Editor/AssetEditor/InputActionTreeView.cs:359)
UnityEditor.IMGUI.Controls.TreeViewController.NotifyListenersThatSelectionChanged () (at C:/buildslave/unity/build/Editor/Mono/GUI/TreeView/TreeViewController.cs:442)
UnityEditor.IMGUI.Controls.TreeViewController.NewSelectionFromUserInteraction (System.Collections.Generic.List`1[T] newSelection, System.Int32 itemID) (at C:/buildslave/unity/build/Editor/Mono/GUI/TreeView/TreeViewController.cs:1201)
UnityEditor.IMGUI.Controls.TreeViewController.SelectionClick (UnityEditor.IMGUI.Controls.TreeViewItem itemClicked, System.Boolean keepMultiSelection) (at C:/buildslave/unity/build/Editor/Mono/GUI/TreeView/TreeViewController.cs:1190)
UnityEditor.IMGUI.Controls.TreeViewController.HandleUnusedMouseEventsForItem (UnityEngine.Rect rect, UnityEditor.IMGUI.Controls.TreeViewItem item, System.Int32 row) (at C:/buildslave/unity/build/Editor/Mono/GUI/TreeView/TreeViewController.cs:404)
UnityEditor.IMGUI.Controls.TreeViewController.DoItemGUI (UnityEditor.IMGUI.Controls.TreeViewItem item, System.Int32 row, System.Single rowWidth, System.Boolean hasFocus) (at C:/buildslave/unity/build/Editor/Mono/GUI/TreeView/TreeViewController.cs:507)
UnityEditor.IMGUI.Controls.TreeViewController.IterateVisibleItems (System.Int32 firstRow, System.Int32 numVisibleRows, System.Single rowWidth, System.Boolean hasFocus) (at C:/buildslave/unity/build/Editor/Mono/GUI/TreeView/TreeViewController.cs:691)
UnityEditor.IMGUI.Controls.TreeViewController.OnGUI (UnityEngine.Rect rect, System.Int32 keyboardControlID) (at C:/buildslave/unity/build/Editor/Mono/GUI/TreeView/TreeViewController.cs:594)
UnityEditor.IMGUI.Controls.TreeView.OnGUI (UnityEngine.Rect rect) (at C:/buildslave/unity/build/Editor/Mono/GUI/TreeView/TreeViewControl/TreeViewControl.cs:414)
UnityEngine.InputSystem.Editor.InputActionTreeView.OnGUI (UnityEngine.Rect rect) (at Library/PackageCache/com.unity.inputsystem@0.9.5-preview/InputSystem/Editor/AssetEditor/InputActionTreeView.cs:1058)
UnityEngine.InputSystem.Editor.InputActionEditorWindow.DrawColumnWithTreeView (UnityEditor.IMGUI.Controls.TreeView treeView, System.Single width, System.Boolean fixedWidth) (at Library/PackageCache/com.unity.inputsystem@0.9.5-preview/InputSystem/Editor/AssetEditor/InputActionEditorWindow.cs:669)
UnityEngine.InputSystem.Editor.InputActionEditorWindow.DrawActionMapsColumn (System.Single width) (at Library/PackageCache/com.unity.inputsystem@0.9.5-preview/InputSystem/Editor/AssetEditor/InputActionEditorWindow.cs:651)
UnityEngine.InputSystem.Editor.InputActionEditorWindow.OnGUI () (at Library/PackageCache/com.unity.inputsystem@0.9.5-preview/InputSystem/Editor/AssetEditor/InputActionEditorWindow.cs:612)
System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at <a8ed250850854b439cedc18931a314fe>:0)
Rethrow as TargetInvocationException: Exception has been thrown by the target of an invocation.
System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at <a8ed250850854b439cedc18931a314fe>:0)
System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) (at <a8ed250850854b439cedc18931a314fe>:0)
UnityEditor.HostView.Invoke (System.String methodName, System.Object obj) (at C:/buildslave/unity/build/Editor/Mono/HostView.cs:359)
UnityEditor.HostView.Invoke (System.String methodName) (at C:/buildslave/unity/build/Editor/Mono/HostView.cs:353)
UnityEditor.HostView.InvokeOnGUI (UnityEngine.Rect onGUIPosition, UnityEngine.Rect viewRect) (at C:/buildslave/unity/build/Editor/Mono/HostView.cs:329)
UnityEditor.DockArea.DrawView (UnityEngine.Rect viewRect, UnityEngine.Rect dockAreaRect, System.Boolean floatingWindow, System.Boolean isBottomTab) (at C:/buildslave/unity/build/Editor/Mono/GUI/DockArea.cs:374)
UnityEditor.DockArea.OldOnGUI () (at C:/buildslave/unity/build/Editor/Mono/GUI/DockArea.cs:341)
UnityEngine.UIElements.IMGUIContainer.DoOnGUI (UnityEngine.Event evt, UnityEngine.Matrix4x4 parentTransform, UnityEngine.Rect clippingRect, System.Boolean isComputingLayout, UnityEngine.Rect layoutSize) (at C:/buildslave/unity/build/Modules/UIElements/IMGUIContainer.cs:298)
UnityEngine.UIElements.IMGUIContainer.HandleIMGUIEvent (UnityEngine.Event e, UnityEngine.Matrix4x4 worldTransform, UnityEngine.Rect clippingRect) (at C:/buildslave/unity/build/Modules/UIElements/IMGUIContainer.cs:483)
UnityEngine.UIElements.IMGUIContainer.HandleIMGUIEvent (UnityEngine.Event e) (at C:/buildslave/unity/build/Modules/UIElements/IMGUIContainer.cs:466)
UnityEngine.UIElements.IMGUIContainer.HandleEvent (UnityEngine.UIElements.EventBase evt) (at C:/buildslave/unity/build/Modules/UIElements/IMGUIContainer.cs:447)
UnityEngine.UIElements.MouseCaptureDispatchingStrategy.DispatchEvent (UnityEngine.UIElements.EventBase evt, UnityEngine.UIElements.IPanel panel) (at C:/buildslave/unity/build/Modules/UIElements/Events/MouseCaptureDispatchingStrategy.cs:93)
UnityEngine.UIElements.EventDispatcher.ProcessEvent (UnityEngine.UIElements.EventBase evt, UnityEngine.UIElements.IPanel panel) (at C:/buildslave/unity/build/Modules/UIElements/EventDispatcher.cs:280)
UnityEngine.UIElements.EventDispatcher.Dispatch (UnityEngine.UIElements.EventBase evt, UnityEngine.UIElements.IPanel panel, UnityEngine.UIElements.DispatchMode dispatchMode) (at C:/buildslave/unity/build/Modules/UIElements/EventDispatcher.cs:156)
UnityEngine.UIElements.BaseVisualElementPanel.SendEvent (UnityEngine.UIElements.EventBase e, UnityEngine.UIElements.DispatchMode dispatchMode) (at C:/buildslave/unity/build/Modules/UIElements/Panel.cs:190)
UnityEngine.UIElements.UIElementsUtility.DoDispatch (UnityEngine.UIElements.BaseVisualElementPanel panel) (at C:/buildslave/unity/build/Modules/UIElements/UIElementsUtility.cs:255)
UnityEngine.UIElements.UIElementsUtility.ProcessEvent (System.Int32 instanceID, System.IntPtr nativeEventPtr) (at C:/buildslave/unity/build/Modules/UIElements/UIElementsUtility.cs:78)
UnityEngine.GUIUtility.ProcessEvent (System.Int32 instanceID, System.IntPtr nativeEventPtr) (at C:/buildslave/unity/build/Modules/IMGUI/GUIUtility.cs:179)
```

**Comment by omid3098** [comment]
Same here:

`Library/PackageCache/com.unity.inputsystem@0.9.5-preview/InputSystem/Editor/AssetEditor/InputActionEditorWindow.cs:452)
UnityEngine.InputSystem.Editor.InputActionTreeView.BuildRoot () (at Library/PackageCache/com.unity.inputsystem@0.9.5-preview/InputSystem/Editor/AssetEditor/InputActionTreeView.cs:103)
UnityEditor.IMGUI.Controls.TreeView+TreeViewControlDataSource.FetchData () (at /home/builduser/buildslave/unity/build/Editor/Mono/GUI/TreeView/TreeViewControl/TreeViewControlDataSource.cs:53)
UnityEditor.IMGUI.Controls.TreeViewDataSource.ReloadData () (at /home/builduser/buildslave/unity/build/Editor/Mono/GUI/TreeView/TreeViewDataSource.cs:53)
UnityEditor.IMGUI.Controls.TreeView+TreeViewControlDataSource.ReloadData () (at /home/builduser/buildslave/unity/build/Editor/Mono/GUI/TreeView/TreeViewControl/TreeViewControlDataSource.cs:25)
UnityEditor.IMGUI.Controls.TreeViewController.ReloadData () (at /home/builduser/buildslave/unity/build/Editor/Mono/GUI/TreeView/TreeViewController.cs:288)
UnityEditor.IMGUI.Controls.TreeView.Reload () (at /home/builduser/buildslave/unity/build/Editor/Mono/GUI/TreeView/TreeViewControl/TreeViewControl.cs:105)
UnityEngine.InputSystem.Editor.InputActionEditorWindow.RebuildActionTree () (at Library/PackageCache/com.unity.inputsystem@0.9.5-preview/InputSystem/Editor/AssetEditor/InputActionEditorWindow.cs:456)
UnityEngine.InputSystem.Editor.InputActionEditorWindow.OnActionMapTreeSelectionChanged () (at Library/PackageCache/com.unity.inputsystem@0.9.5-preview/InputSystem/Editor/AssetEditor/InputActionEditorWindow.cs:425)
UnityEngine.InputSystem.Editor.InputActionTreeView.SelectionChanged (System.Collections.Generic.IList`1[T] selectedIds) (at Library/PackageCache/com.unity.inputsystem@0.9.5-preview/InputSystem/Editor/AssetEditor/InputActionTreeView.cs:359)
UnityEditor.IMGUI.Controls.TreeViewController.NotifyListenersThatSelectionChanged () (at /home/builduser/buildslave/unity/build/Editor/Mono/GUI/TreeView/TreeViewController.cs:442)
UnityEditor.IMGUI.Controls.TreeViewController.NewSelectionFromUserInteraction (System.Collections.Generic.List`1[T] newSelection, System.Int32 itemID) (at /home/builduser/buildslave/unity/build/Editor/Mono/GUI/TreeView/TreeViewController.cs:1201)
UnityEditor.IMGUI.Controls.TreeViewController.SelectionClick (UnityEditor.IMGUI.Controls.TreeViewItem itemClicked, System.Boolean keepMultiSelection) (at /home/builduser/buildslave/unity/build/Editor/Mono/GUI/TreeView/TreeViewController.cs:1190)
UnityEditor.IMGUI.Controls.TreeViewController.HandleUnusedMouseEventsForItem (UnityEngine.Rect rect, UnityEditor.IMGUI.Controls.TreeViewItem item, System.Int32 row) (at /home/builduser/buildslave/unity/build/Editor/Mono/GUI/TreeView/TreeViewController.cs:404)
UnityEditor.IMGUI.Controls.TreeViewController.DoItemGUI (UnityEditor.IMGUI.Controls.TreeViewItem item, System.Int32 row, System.Single rowWidth, System.Boolean hasFocus) (at /home/builduser/buildslave/unity/build/Editor/Mono/GUI/TreeView/TreeViewController.cs:507)
UnityEditor.IMGUI.Controls.TreeViewController.IterateVisibleItems (System.Int32 firstRow, System.Int32 numVisibleRows, System.Single rowWidth, System.Boolean hasFocus) (at /home/builduser/buildslave/unity/build/Editor/Mono/GUI/TreeView/TreeViewController.cs:691)
UnityEditor.IMGUI.Controls.TreeViewController.OnGUI (UnityEngine.Rect rect, System.Int32 keyboardControlID) (at /home/builduser/buildslave/unity/build/Editor/Mono/GUI/TreeView/TreeViewController.cs:594)
UnityEditor.IMGUI.Controls.TreeView.OnGUI (UnityEngine.Rect rect) (at /home/builduser/buildslave/unity/build/Editor/Mono/GUI/TreeView/TreeViewControl/TreeViewControl.cs:414)
UnityEngine.InputSystem.Editor.InputActionTreeView.OnGUI (UnityEngine.Rect rect) (at Library/PackageCache/com.unity.inputsystem@0.9.5-preview/InputSystem/Editor/AssetEditor/InputActionTreeView.cs:1058)
UnityEngine.InputSystem.Editor.InputActionEditorWindow.DrawColumnWithTreeView (UnityEditor.IMGUI.Controls.TreeView treeView, System.Single width, System.Boolean fixedWidth) (at Library/PackageCache/com.unity.inputsystem@0.9.5-preview/InputSystem/Editor/AssetEditor/InputActionEditorWindow.cs:669)
UnityEngine.InputSystem.Editor.InputActionEditorWindow.DrawActionMapsColumn (System.Single width) (at Library/PackageCache/com.unity.inputsystem@0.9.5-preview/InputSystem/Editor/AssetEditor/InputActionEditorWindow.cs:651)
UnityEngine.InputSystem.Editor.InputActionEditorWindow.OnGUI () (at Library/PackageCache/com.unity.inputsystem@0.9.5-preview/InputSystem/Editor/AssetEditor/InputActionEditorWindow.cs:612)
System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at <599589bf4ce248909b8a14cbe4a2034e>:0)
Rethrow as TargetInvocationException: Exception has been thrown by the target of an invocation.
System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at <599589bf4ce248909b8a14cbe4a2034e>:0)
System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) (at <599589bf4ce248909b8a14cbe4a2034e>:0)
UnityEditor.HostView.Invoke (System.String methodName, System.Object obj) (at /home/builduser/buildslave/unity/build/Editor/Mono/HostView.cs:359)
UnityEditor.HostView.Invoke (System.String methodName) (at /home/builduser/buildslave/unity/build/Editor/Mono/HostView.cs:353)
UnityEditor.HostView.InvokeOnGUI (UnityEngine.Rect onGUIPosition, UnityEngine.Rect viewRect) (at /home/builduser/buildslave/unity/build/Editor/Mono/HostView.cs:329)
UnityEditor.DockArea.DrawView (UnityEngine.Rect viewRect, UnityEngine.Rect dockAreaRect, System.Boolean floatingWindow, System.Boolean isBottomTab) (at /home/builduser/buildslave/unity/build/Editor/Mono/GUI/DockArea.cs:374)
UnityEditor.DockArea.OldOnGUI () (at /home/builduser/buildslave/unity/build/Editor/Mono/GUI/DockArea.cs:341)
UnityEngine.UIElements.IMGUIContainer.DoOnGUI (UnityEngine.Event evt, UnityEngine.Matrix4x4 parentTransform, UnityEngine.Rect clippingRect, System.Boolean isComputingLayout, UnityEngine.Rect layoutSize) (at /home/builduser/buildslave/unity/build/Modules/UIElements/IMGUIContainer.cs:298)
UnityEngine.UIElements.IMGUIContainer.HandleIMGUIEvent (UnityEngine.Event e, UnityEngine.Matrix4x4 worldTransform, UnityEngine.Rect clippingRect) (at /home/builduser/buildslave/unity/build/Modules/UIElements/IMGUIContainer.cs:483)
UnityEngine.UIElements.IMGUIContainer.HandleIMGUIEvent (UnityEngine.Event e) (at /home/builduser/buildslave/unity/build/Modules/UIElements/IMGUIContainer.cs:466)
UnityEngine.UIElements.IMGUIContainer.HandleEvent (UnityEngine.UIElements.EventBase evt) (at /home/builduser/buildslave/unity/build/Modules/UIElements/IMGUIContainer.cs:447)
UnityEngine.UIElements.MouseCaptureDispatchingStrategy.DispatchEvent (UnityEngine.UIElements.EventBase evt, UnityEngine.UIElements.IPanel panel) (at /home/builduser/buildslave/unity/build/Modules/UIElements/Events/MouseCaptureDispatchingStrategy.cs:93)
UnityEngine.UIElements.EventDispatcher.ProcessEvent (UnityEngine.UIElements.EventBase evt, UnityEngine.UIElements.IPanel panel) (at /home/builduser/buildslave/unity/build/Modules/UIElements/EventDispatcher.cs:280)
UnityEngine.UIElements.EventDispatcher.Dispatch (UnityEngine.UIElements.EventBase evt, UnityEngine.UIElements.IPanel panel, UnityEngine.UIElements.DispatchMode dispatchMode) (at /home/builduser/buildslave/unity/build/Modules/UIElements/EventDispatcher.cs:156)
UnityEngine.UIElements.BaseVisualElementPanel.SendEvent (UnityEngine.UIElements.EventBase e, UnityEngine.UIElements.DispatchMode dispatchMode) (at /home/builduser/buildslave/unity/build/Modules/UIElements/Panel.cs:190)
UnityEngine.UIElements.UIElementsUtility.DoDispatch (UnityEngine.UIElements.BaseVisualElementPanel panel) (at /home/builduser/buildslave/unity/build/Modules/UIElements/UIElementsUtility.cs:255)
UnityEngine.UIElements.UIElementsUtility.ProcessEvent (System.Int32 instanceID, System.IntPtr nativeEventPtr) (at /home/builduser/buildslave/unity/build/Modules/UIElements/UIElementsUtility.cs:78)
UnityEngine.GUIUtility.ProcessEvent (System.Int32 instanceID, System.IntPtr nativeEventPtr) (at /home/builduser/buildslave/unity/build/Modules/IMGUI/GUIUtility.cs:179)`

Edit: 
Platform: Linux x64
Unity: 2019.2.3f1
Input system: 0.9.5

**Comment by EricBoiseLGSVL** [comment]
Having this issue also.
```
LayoutNotFoundException: Cannot find control layout 'HID::Logitech G920 Driving Force Racing Wheel for Xbox One Joystick'
UnityEngine.InputSystem.Layouts.InputControlLayout+Cache.FindOrLoadLayout (System.String name) (at Library/PackageCache/com.unity.inputsystem@1.0.0-preview/InputSystem/Controls/InputControlLayout.cs:2028)
```

**Comment by jakobkilian** [comment]
I had the same issue as EricBoiseLGSVL. 
Input Manager didn't work at all: Couldn't select devices in Project Settings, Input Debugger and the .inputactions file is blank. 
But it seems to be a bug: the issue just occurs, while the G29 is not connected. So I can just work on my project with the driving wheel connected. 
Any suggestions?

**Comment by LuisAntonCanalis** [comment]
I just experienced this. I opened the project on a computer where certain gamepad is not conected. Got 'LayoutNotFoundException: Cannot find control layout ...' (and the name of the device) and the action map in the inputactions file related to player control is empty. So:
- opened the project in computer A, where I tested the game with to gamepads. Saved, pushed to repo.
- pulled and opened the project in computer B, where those gamepads are not connected. Action map in inputactions file is empty and get the LayoutNotFoundException: Cannot find control layout  error.
